RefactoringEssentials/CSharp/Converter/MethodBodyVisitor.cs

```diff
@@ -16,20 +16,30 @@ class MethodBodyVisitor : VBasic.VisualBasicSyntaxVisitor<SyntaxList<StatementSy
 		{
 			SemanticModel semanticModel;
 			NodesVisitor nodesVisitor;
+			private readonly Stack<string> withBlockTempVariableNames;
 
             public bool IsIterator { get; set; }
 
-			public MethodBodyVisitor(SemanticModel semanticModel, NodesVisitor nodesVisitor)
+			public MethodBodyVisitor(SemanticModel semanticModel, NodesVisitor nodesVisitor, Stack<string> withBlockTempVariableNames)
 			{
 				this.semanticModel = semanticModel;
 				this.nodesVisitor = nodesVisitor;
+				this.withBlockTempVariableNames = withBlockTempVariableNames;
 			}
 
 			public override SyntaxList<StatementSyntax> DefaultVisit(SyntaxNode node)
 			{
 				throw new NotImplementedException(node.GetType() + " not implemented!");
 			}
 
+			public override SyntaxList<StatementSyntax> VisitStopOrEndStatement(VBSyntax.StopOrEndStatementSyntax node)
+			{
+				var cSharpEquivalent = node.StopOrEndKeyword.IsKind(VBasic.SyntaxKind.StopKeyword) ? "System.Diagnostics.Debugger.Break();"
```

Comment:
Sorry for nitpicking, but I think it would be much better to use normal if statements here instead of if expressions, it's kinda hard to read...